src/components/Hero.js

```diff
@@ -71,14 +71,14 @@ function Hero({handleLogIn}) {
           <Flex className="quotes">
             <Flex className="quote">
               <img alt="open sauced" src={gucci} />
-              <span>
+              <span style={{fontSize: 18}}>
```

Comment:
Not need, per my comment on the PR. Instead, use extra padding. Be sure to use the variabls.js. 

```css
.quotes {
  padding-bottom :${size.tiny}
  ...
}
```